media_driver/cmake/linux/media_gen_flags_linux.cmake

```diff
@@ -81,6 +81,10 @@ cmake_dependent_option(GEN12_ADLP
     "Enabled ADLP support (Gen12)" ON
     "GEN12_TGLLP" OFF)
 
+cmake_dependent_option(GEN12_ADLN
```

Comment:
I think , maybe the pre-condition should not be GEN12, it should be GEN12_TGLLP, because ADLN are using lots of TGLLP code

Reply:
This commit was already referring to  IGFX_GEN12_ADLN_SUPPORTED
 https://github.com/intel/media-driver/commit/348e98e90d240deecd3040f3846a27e9bb6c3ac1
 
The media_driver/media_interface/media_srcs.cmake ,  was using GEN12_ADLN 

let me